CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.7.0](https://github.com/sonata-project/SonataIntlBundle/compare/2.6.0...2.7.0) - 2019-08-08
+
+### Fixed
+- `DateTimeExtension::getFilters()` returning a collection of `TwigFunction` instead of `TwigFilter`.
```

Comment:
IMO this entry has no sense since the fixed issue is not present in a previous release. See https://github.com/sonata-project/SonataIntlBundle/pull/264#issuecomment-508487392.